pioneer/packages/page-accounts/src/Accounts/PasswordInput.tsx

```diff
@@ -17,13 +17,11 @@ type Props = {
 
 export default function PasswordInput ({ onChange, onEnter, password }: Props): React.ReactElement {
   const { t } = useTranslation();
-  const [isPassValid, setPassValid] = useState<boolean>(false);
-  const [{ isPass2Valid, password2 }, setPassword2] = useState({ isPass2Valid: false, password2: '' });
+  const [isPassValid, setPassValid] = useState<boolean>(true);
+  const [{ isPass2Valid, password2 }, setPassword2] = useState({ isPass2Valid: true, password2: '' });
 
   const _onPasswordChange = useCallback(
     (password: string) => {
-      const isPassValid = keyring.isPassValid(password);
```

Comment:
Just noticed we have Joystream-specific `isPasswordValid` function inside `joy-utils/src/functions/accounts.ts`.
It may be cleaner/safer to use of that one, because:
1. It preserves the `keyring.isPassValid` check (I know that currently it just checks if length > 0, but it's possible that in some future version it will also include some other security checks we may need)
2. This way we can add our custom rules to password validation later and will only have to change this in one place.
3. If we decide to pull from upstream again this should give us a cleaner indication of what was the reason behind this change.